### PR TITLE
GCP KMS many fixes

### PR DESCRIPTION
**Pull request by Alevsk**
- Operator-UI didnt correctly append gcp kms configuration

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx [hunk 1]
```diff
@@ -2375,6 +2366,20 @@ const AddTenant = ({ classes }: IAddTenantProps) => {
...
+                  {/*      setGcpPrivateKeyVal(fileName);*/}
+                  {/*      clearValidationError("gcp_private_key");*/}
+                  {/*    }}*/}
+                  {/*    accept=".key,.pem"*/}
+                  {/*    id="gcp_private_key"*/}
+                  {/*    name="gcp_private_key"*/}
+                  {/*    label="Private Key"*/}
+                  {/*    value={gcpPrivateKeyVal}*/}
+                  {/*  />*/}
+                  {/*</Grid>*/}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx:2382]: Is it ok to have this  commented?

File: portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx [hunk 2]
```diff
@@ -299,6 +296,7 @@ const AddTenant = ({ classes }: IAddTenantProps) => {
   const [vaultCertVal, setVaultCertVal] = useState<string>("");
   const [vaultCAVal, setVaultCAVal] = useState<string>("");
   const [gemaltoCAVal, setGemaltoCAVal] = useState<string>("");
+  const [gcpPrivateKeyVal, setGcpPrivateKeyVal] = useState<string>("");
```
Comment (cesnietor) [inline portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx:299]: where is this being used?

File: portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx [hunk 3]
```diff
@@ -1111,6 +1098,7 @@ const AddTenant = ({ classes }: IAddTenantProps) => {
                     client_id: gcpClientID,
                     private_key_id: gcpPrivateKeyID,
                     private_key: gcpPrivateKey,
```
Comment (cesnietor) [inline portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx:1100]: where is gcpPrivateKey being set?

Comment (Alevsk) [reply portal-ui/src/screens/Console/Tenants/ListTenants/AddTenant.tsx]: line `2355`

**Review by bexsoft — APPROVED** [review]

**Review by cesnietor — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**